Repost Item Valuation | test_deduplication | status error

### Information about bug

    self.assertEqual(doc.status, status)
AssertionError: 'Queued' != 'Skipped'
- Queued
+ Skipped

### Module

stock

### Version

Frappe Framework: v15.51.2
ERPNext: v15.50.

### Installation method

manual install

### Relevant log output / Stack trace / Full Error Message.

```shell
Traceback (most recent call last):
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/stock/doctype/repost_item_valuation/test_repost_item_valuation.py", line 159, in test_deduplication
    _assert_status(riv2, "Skipped")
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/stock/doctype/repost_item_valuation/test_repost_item_valuation.py", line 133, in _assert_status
    self.assertEqual(doc.status, status)
AssertionError: 'Queued' != 'Skipped'
- Queued
+ Skipped
```

--- a/erpnext/stock/doctype/repost_item_valuation/repost_item_valuation.py
+++ b/erpnext/stock/doctype/repost_item_valuation/repost_item_valuation.py
@@ -242,8 +242,7 @@ class RepostItemValuation(Document):
 			WHERE item_code = %(item_code)s
 				AND warehouse = %(warehouse)s
 				AND name != %(name)s
-				AND posting_date > %(posting_date)s
-				AND posting_time > %(posting_time)s
+				AND CONCAT(posting_date, ' ', posting_time) > CONCAT(%(posting_date)s, ' ', %(posting_time)s)
 				AND docstatus = 1
 				AND status = 'Queued'
 				AND based_on = 'Item and Warehouse'

